lib/sundials-config/prepare.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+SUNDIALS_VERSION="4.0.1"
+
+SUNDIALS_TAR=sundials-${SUNDIALS_VERSION}.tar.gz
+
+tar xvzf $SUNDIALS_TAR
+mv sundials-${SUNDIALS_VERSION} sundials_${SUNDIALS_VERSION}
+
+cd sundials_${SUNDIALS_VERSION}
+
+rm -rf INSTALL_GUIDE.pdf config/ doc/ examples/ test/ */cvode */ida */arkode */kinsol
+find . -name CMakeLists.txt -exec rm {} \;
+
+find src -name "*.c" -type f -exec sed -E -i _orig  's#[^sf]printf\(#STAN_SUNDIALS_PRINTF(#'g {} \;
+find src -name "*.c" -type f -exec sed -E -i _orig  's#fprintf\(#STAN_SUNDIALS_FPRINTF(#'g {} \;
+
```

Comment:
remove this whitespace -- these 3 lines go together. Adding the single line makes it seem like the next line is different.